cfme/automate/__init__.py

```diff
@@ -71,3 +71,22 @@ class AutomateExplorer(CFMENavigateStep):
 
     def step(self):
         self.view.navigation.select('Automate', 'Explorer')
+
+
+class AutomateSimulationView(BaseLoggedInPage):
+    @property
+    def is_displayed(self):
+        return (
+            self.logged_in_as_current_user and
+            self.navigation.currently_selected == ['Automate', 'Simulation'])
+
+    # TODO: Actually convert this to Widgetastic.
+
+
+@navigator.register(Server)
```

Comment:
The only problem with this is that we need to import cfme.automate in order to gain access to this. Is say this because we don't explicitly have an object in the simulation page, normally, we need to load an object and by loading the object, we MUST import that module.